modules/030-cloud-provider-azure/cloud-instance-manager/machine-class.yaml

```diff
@@ -43,3 +43,5 @@ spec:
             path: /home/azureuser/.ssh/authorized_keys
             keyData: {{ .Values.nodeManager.internal.cloudProvider.azure.sshPublicKey }}
     zone: {{ .zoneName }}
+    networkProfile:
+      acceleratedNetworking: {{ .nodeGroup.instanceClass.acceleratedNetworking }}
```

Comment:
`acceleratedNetworking` must be added to MachineClass checksum

Reply:
done